app/src/main/res/layout/main_app_bar.xml

```diff
@@ -25,7 +25,10 @@
             android:id="@+id/fragment_container"
             android:layout_width="match_parent"
             android:layout_height="match_parent"
-            app:layout_behavior="@string/appbar_scrolling_view_behavior" />
+            app:layout_behavior="@string/appbar_scrolling_view_behavior"
+            android:name="android.navigation.fragment.NavHostFragment"
+            app:navGraph="@navigation/main_navigation"
+            app:defaultNavHost="true"/>
```

Comment:
This should be in AuthActivity as nav graph of Auth will be different from main nav

Reply:
@iamareebjamal I already tried incorporating the app_bar code in auth_activity but still the error persists.